### DISCUSSION TITLE
续费两年后 专业版变成普通版

### DISCUSSION THREAD

**Discussion by lisheng811** [getlantern/lantern-forum-cn#487] [General]
续费两年后 专业版变成普通版
版本： 7.8.2-windows7 (20240305.210652)
账号： lisheng811@outlook.com
发了邮件 没有解决
请处理

**Comment by MaxMKMax** [comment]
@lisheng811 您好! 我这里直接帮您对接负责人. 这样快一点. 请您也提供我付款单据. 谢谢哦

**Reply by lisheng811** [reply]
![SmartSelect_20240506_014721_Alipay](https://github.com/getlantern/lantern-forum-cn/assets/23717125/919ba41e-14a5-4fd3-be16-1fb17e1da964)

**Comment by lisheng811** [comment]
![SmartSelect_20240506_014721_Alipay](https://github.com/getlantern/lantern-forum-cn/assets/23717125/de237cf5-1a49-424b-9753-d38d593a634f)

**Comment by lisheng811** [comment]
Mike K ***@***.***> 于2024年5月5日周日 23:47写道：

**Reply by MaxMKMax** [reply]
@lisheng811 好的. 谢谢. 我帮您跟进一下. 请给我们几天的时间. 我们会尽快. 谢谢.

**Comment by MaxMKMax** [comment]
@lisheng811 您好. 已经给您发激活码了. 请您查看邮箱. 来自于 mike@getlantern.org

**Reply by lisheng811** [reply]
激活码收到用上了 恢复了专业版 但一直不通
我的系统是win7 lantern是7.8.2-windows7 (20240305.210652)
不通是因为win7吗

**Reply by lisheng811** [reply]
不好意思 没有注意到重装lantern后端口号变了 
修改设置后 通了
请问 7.8.2-windows7 (20240305.210652)后还支持win7吗
